File: December2021/MinCostToMoveChipsToSamePosition.java
class Solution {
	public int minCostToMoveChips(int[] position) {
		int even = 0, odd = 0;

		for(int p : position){
			if(p%2==0) even++;
			else odd++;
		}

		return Math.min(even,odd);
	}
}

File: December2021/C++/Min_Cost_to_move_chips_to_same_position.cpp
class Solution {
public:
    int minCostToMoveChips(vector<int>& position) {
        int even=0,odd=0;
        for(auto& i:position)
        {
            if (i%2) ++odd;
            else ++even;
        }
        return min(odd,even);
    }
};